addusers.py

```diff
@@ -378,7 +430,9 @@ def parse_rows(rows, select_user=None):
     rows = sheet.get_all_rows("Form Responses 1")
 
     try:
-        content, bad_rows = parse_rows(rows, select_user=args.user)
+        content, bad_rows = parse_rows(rows, select_user=args.user,
```

Comment:
Instead, of passing the args in, check at line 430 to see if you are supposed to get manual input or spreadsheet input, and create `rows` appropriately.  Then just pass in `rows`.   That way you are not trying to read the spreadsheet when you don't need to.